app/views/templates/error_template.scala.html

```diff
@@ -29,7 +29,7 @@ <h1>@heading</h1>
 
 
 @{
-    templateRenderer.renderDefaultTemplate(layouts.article(mainContent), Map[String, Any](
+    templateRenderer.renderDefaultTemplate(config.ApplicationConfig.frontendTemplatePath, layouts.article(mainContent), Map[String, Any](
```

Comment:
appConfig is being used for some reason, may as well use it here

Reply:
done